app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -65,4 +70,36 @@
 
     <%= f.continue %>
   <% end %>
-<% end %>
+  <script>
+    document.addEventListener("DOMContentLoaded", function() {
```

Comment:
[pebble] Seeing that we're using the same logic twice, I wonder if it's worth adding the function passed to the event listener to `app/javascript. You could refactor it to work for both cases by passing in the IDs of all not-none checkboxes, for each id adding the checkbox to the checkbox array if they're not null, passing in the noneCheckbox ID, and then optional checkbox ids that should be hidden when none is checked

Reply:
i like this suggestion too. we or other states might need this pattern in more places so it would be handy to reuse